python/kserve/kserve/protocol/dataplane.py

```diff
@@ -360,6 +364,9 @@ async def explain(
         """
         # call model locally or remote model workers
         model = self.get_model(model_name)
+        if isinstance(model, OpenAIModel):
+            error_msg = f"Model {model_name} is of type OpenAIModel. It does not support the explain method."
+            raise InvalidInput(reason=error_msg)
```

Comment:
I am not sure about this, but openai does not support explanation so maybe we should leave this out, do want to block people who wants to implement this for `OpenAIModel`